CHANGELOG.rst

```diff
@@ -9,6 +9,13 @@ Documentation
 ~~~~~~~~~~~~~
 - Added more info in ``quick-start.ipynb`` (:pr:`316`) `Ray Bell`_.
 
+Breaking changes
+~~~~~~~~~~~~~~~~
+- Renamed `mae_test` to `significance_test` to make this comparative metric
+  generic. Now, it accepts any of the distance metrics functions except for
+  `mape`. The new function has an additional argument called `metric` which is a
+  string and name of the target distance metric, . (:pr:`319`)
```

Comment:
don't forget to add your name here (and at the bottom). I think you'll also have to add to https://github.com/cheginit/xskillscore/blob/main/docs/source/contributors.rst